parsers/imageAltTag.js

```diff
@@ -0,0 +1,47 @@
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('parser:imageAltTag');
+var parse = require('./lib/parse');
+
+/*
+ * Extract alt text from img tags
+ */
+
+var PARSER_COMPLETED_SUCCESSFULLY = false;
+var FACEBOOK_ALT_TEXT_START = 'Image may contain:';
+
```

Comment:
so I'll suggest to take all the `alt text` we can find, and let the user judge if is about a source attribution (like Portuguese) or an Artificial Intelligence interpretation of content (English, Italian, and I don't know which others)